.env.example

```diff
@@ -8,3 +8,9 @@ DATABASE_URL=postgres://postgres:@localhost:5432/compass
 # Only needed if you want to use sign in with Google
 # GOOGLE_CLIENT_ID=
 # GOOGLE_CLIENT_SECRET=
+
+S3_ENDPOINT=http://localhost:9000
+S3_REGION=us-east-1
```

Comment:
in this specific case, I labeled them with a `S3` prefix instead of `AWS` since there are other available backends for S3 (as explained in the PR OP).

Reply:
SG. I would recommend s/S3_BUCKET_NAME/FILE_UPLOAD_S3_BUCKET or similar and s/compass/compass_files below on line 16 to clarify the purpose of this bucket, and to differentiate it from other buckets we will use in Compass, e.g. #87 uses two more buckets - one for Terraform state and another for env files.

Reply:
65cc3d7